libprecompiled/extension/wedpr-generated/asset_hiding/ffi_hidden_asset.h

```diff
@@ -0,0 +1,44 @@
+#include <cstdarg>
+#include <cstdint>
+#include <cstdlib>
+#include <new>
+
+extern "C" {
+
+char* get_credit_storage_by_fulfill_argument(char* fulfill_argument_pb);
+
```

Comment:
Remove emtpy file here.
Add a TODO to add comment later for those public FFI API

Reply:
The file is generated.